deploy/foreman-tasks.sysconfig

```diff
@@ -12,3 +12,15 @@ RUBY_GC_MALLOC_LIMIT_MAX=16000100
 RUBY_GC_MALLOC_LIMIT_GROWTH_FACTOR=1.1
 RUBY_GC_OLDMALLOC_LIMIT=16000100
 RUBY_GC_OLDMALLOC_LIMIT_MAX=16000100
+
+#Set the number of executors you want to run
+#EXECUTORS_COUNT=1
+
+#Set memory limit for executor process, before it's restarted automatically
+#EXECUTOR_MEMORY_LIMIT=1gb
```

Comment:
From what I've seen in production, suggesting 2GB might be slightly better default